docs/how_to_guides/configuring_metadata_stores/how_to_configure_a_validation_result_store_in_s3.rst

```diff
@@ -3,16 +3,85 @@
 How to configure a Validation Result store in S3
 ================================================
 
-.. admonition:: Admonition from Mr. Dickens
+This guide will help you configure a new storage location for Validations on Amazon S3.
 
-    "Whether I shall turn out to be the hero of my own life, or whether that station will be held by anybody else, these pages must show."
+.. admonition:: Prerequisites: This how-to guide assumes that you have already:
 
+    - Configured a :ref:`Data Context <tutorials__getting_started__initialize_a_data_context>`.
+    - Configured an :ref:`Expectation Suite <tutorials__getting_started__create_your_first_expectations>`.
+    - Installed `boto3 <https://github.com/boto/boto3>`_ in the environment where you will run Great Expectations.
+    - Identified the S3 bucket and prefix where Validation results will be stored.
 
-This guide is a stub. We all know that it will be useful, but no one has made time to write it yet.
+Steps
+-----
+1. **Identify your Data Context Validations Store**
 
-If it would be useful to you, please comment with a +1 and feel free to add any suggestions or questions below.
+    Look for the following section in your Data Context's ``great_expectations.yml`` file:
+
+    .. code-block:: yaml
+
+        validations_store_name: validations_store
+
+        stores:
+            validations_store:
+                class_name: ValidationsStore
+                store_backend:
+                    class_name: TupleFilesystemStoreBackend
+                    base_directory: uncommitted/validations/
+
+    The configuration file tells Great Expectations to look for Validations in a store called ``validations_store``. It also creates a ``ValidationsStore`` called ``validations_store`` that is backed by a Filesystem and will store validations under the ``base_directory`` ``uncommitted/validations`` (the default).
+
+
+2. **Update your configuration file to include a new store for Validation results on S3.**
+
+    In the example below, the new store's name is set to ``validations_S3_store``, but it can be any name you like.  We also need to make some changes to the ``store_backend`` settings.  The ``class_name`` will be set to ``TupleS3StoreBackend``, ``bucket`` will be set to the address of your S3 bucket, and ``prefix`` will be set to the folder in your S3 bucket where Validation results will be located.  If you are also storing :ref:`Expectations in S3, <how_to_guides__configuring_metadata_stores__how_to_configure_an_expectation_store_in_amazon_s3>` please ensure that the ``prefix`` values are disjoint and one is not a substring of the other.
```

Comment:
```suggestion
    In the example below, the new store's name is set to ``validations_S3_store``, but it can be any name you like.  We also need to make some changes to the ``store_backend`` settings.  The ``class_name`` will be set to ``TupleS3StoreBackend``, ``bucket`` will be set to the address of your S3 bucket, and ``prefix`` will be set to the folder in your S3 bucket where Validation results will be located.
    
.. warning::

    If you are also storing :ref:`Expectations in S3, <how_to_guides__configuring_metadata_stores__how_to_configure_an_expectation_store_in_amazon_s3>` using the same bucket, please ensure that the ``prefix`` values are disjoint and one is not a substring of the other. Otherwise, your Expectations and Validation Results Stores will not be able to identify which JSON files contain Expectations versus Validation Results.
```